Makefile

```diff
@@ -4,6 +4,20 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 NAMESPACE ?= nmstate
 
+NMSTATE_CURRENT_COPR_REPO=nmstate-0.3
+NM_CURRENT_COPR_REPO=NetworkManager-1.26
+
+NMSTATE_FUTURE_COPR_REPO=nmstate
```

Comment:
maybe worth to allow overriding those argument from cli ?

Reply:
We don't want that, we just want to put the release on Makefile and open the door to test know future and current.